components/dashboard/src/App.tsx

```diff
@@ -119,6 +119,28 @@ function isWebsiteSlug(pathName: string) {
     return slugs.some((slug) => pathName.startsWith("/" + slug + "/") || pathName === "/" + slug);
 }
 
+// A wrapper for <Route> that redirects to the workspaces screen if the user isn't a admin.
+// This wrapper only accepts the component property
+function AdminRoute({ component }: any) {
```

Comment:
It's awesome that you found my suggestion useful 🥳 

Shouldn't this component also get the other props from `Route`?
In the original example they do it like this with `...rest`:

```tsx
function PrivateRoute({ children, ...rest }) {
  let auth = useAuth();
  return (
    <Route
      {...rest}
```

Reply:
Just want to be restrictive here, as we know we only send `component` here based on the usages in the admin routes.

We can always do `...rest` if we add something at the references! :)

Reply:
Makes sense :) but how does it work without even just the `path` prop?

Reply:
Good question! probably because we don't use `path` anywhere lower down! It's used with higher level Routers but not here, maybe?